config_defaults.yml

```
@@ -32,7 +32,7 @@ Gemfile:
         ruby-version: '2.5'
         ruby-operator: '>='
       - gem: voxpupuli-release
-        version: '>= 1.2.0'
+        version: '>= 2.0.0'
```

Shall we pin to semver, just like `voxpupuli-test` and `voxpupuli-acceptance`?
```suggestion
        version: '~> 2.0'
```

---

yes we should do that.

---

Done!